test.js

```diff
@@ -30,7 +30,6 @@ test('isStream.writable()', t => {
 	t.false(isStream.writable(new Stream.Stream()));
 	t.false(isStream.writable(new Stream.Readable()));
 	t.false(isStream.writable(fs.createReadStream('test.js')));
-	t.false(isStream.writable(new net.Socket()));
```

Comment:
?

Reply:
I removed this test because :
```js
// in node < 14 the test PASS
t.false(isStream.writable(new net.Socket()));
t.false(isStream.readable(new net.Socket()));

// in node >= 14 the test FAILS
t.false(isStream.writable(new net.Socket()));
t.false(isStream.readable(new net.Socket()));
```
![image](https://user-images.githubusercontent.com/1759811/89126260-b145e000-d4e4-11ea-9bb0-e0c6040526db.png)

Any suggestions for how to keep them and make the tests pass in node versions >= 14 and < 14 ?

Reply:
The assertions should still be kept though, as they indicate something about Node.js 14 or the checks are broken. They could be commented out for now though.

Reply:
Ok. 203d9d7